cmd/crictl/image.go

```diff
@@ -270,6 +266,21 @@ func getAuth(creds string) (*pb.AuthConfig, error) {
 	}, nil
 }
 
+// Support these cases as below:
```

Comment:
I think it's fine to assume that this should always be `image:tag` or `image:digest`.

Reply:
Sorry, what do you mean?

Reply:
I think if the repotag is not `image:tag`, we could log an error. Ideally repo tag should always be `image:tag`.

Reply:
Do you mean the cri-containerd or docker-shim should return "image:tag"?
In any case crictl should not panic. Crictl need to protect self.

Reply:
@yanxuean In which case repoTags will be `:tag2` or `tag1:`?

Reply:
I don't know :)
I only protect self.